apps/pie-storybook/stories/testing/pie-list.test.stories.ts

```diff
@@ -25,10 +52,44 @@ const listStoryMeta: ListStoryMeta = {
 
 export default listStoryMeta;
 
-// TODO: remove the eslint-disable rule when props are added
-// eslint-disable-next-line no-empty-pattern
-const Template = ({}: ListProps) => html`
-    <pie-list></pie-list>
+const Template = ({ variant, hasDividers, listType }: ListProps) => html`
+    <pie-list
+        .variant=${variant}
+        .listType=${listType}
+        .hasDividers=${hasDividers}>
```

Comment:
the dot syntax is needed for non primitive values such as arrays or objects or if we need to set it via js. is there a reason why we are using that? otherwise i suggest to use


```suggestion
        variant=${ifDefined(variant)}
        listType=${ifDefined(listType)}
        ?hasDividers=${hasDividers}>
```